api/base.py

```
@@ -942,167 +1007,386 @@ def fetch_response_with_retry():
-                    res_list = multi_cut(res, _ORIGIN_HTML_CONTENT)
-                    if res_list is not None and options_list is not None:
-                        for _a in clean_res(res_list):
-                            matched = False
+            _ORIGIN_HTML_CONTENT = final_resp.text  # 用于配合输出网页源码, 帮助修复#391错误
+
+            # 2. 设置上一轮错误反馈（供AI重新作答时参考）
+            if feedback_history and hasattr(self.tiku, 'set_work_feedback'):
+                try:
+                    self.tiku.set_work_feedback(feedback_history)
+                    logger.debug("已将上一轮错误反馈设置到题库")
+                except Exception as e:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)